src/main/java/com/conveyal/r5/profile/PerTargetPropagater.java

```diff
@@ -0,0 +1,107 @@
+package com.conveyal.r5.profile;
+
+import com.conveyal.r5.streets.LinkedPointSet;
+import gnu.trove.map.TIntIntMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+
+/**
+ * This is similar to the Propagater class, but instead of the reducer function being called with each iteration,
+ * it is called with the travel times to a particular target for all iterations.
+ *
+ * The reducer will be called with the target index, and a boolean array of whether that target is reachable at each iteration.
```

Comment:
Clarify: what is a "target index"? A 1D offset into an array representing a 2D grid of cells?